features/polyExplorer/src/context/explorer-context.jsx

```diff
@@ -143,13 +143,13 @@ export const ExplorerProvider = ({ children }) => {
         }
     }
 
-    function createPopUp({ type, content }) {
+    function createPopUp({ type, content, props, onClose }) {
         // This is a temporary fix - when the HTRT is not full size anymore it should not change the title any longer
         if (type.endsWith("-info"))
             pod.polyNav.setTitle(i18n.t(`common:screenTitle.how.to.read.this`));
         if (type == "info-main")
             pod.polyNav.setTitle(i18n.t(`common:screenTitle.info`));
-        setPopUp({ component: popUps[type], content });
+        setPopUp({ component: popUps[type], content, onClose, props });
```

Comment:
Er. This swap in order might lead to some confusion. Is it possible to keep it the same everywhere? They're keys, after all...